node/Makefile

```diff
@@ -365,12 +365,18 @@ load-container-images: $(K8ST_IMAGE_TARS) $(KUBECTL)
 
 .PHONY: st-checks
 st-checks:
```

Comment:
This target wasn't being used and I think it was out of date so I repurposed it

Reply:
it was used on line 412 below, right? Shouldn't we keep those checks and preps? Idk

Reply:
It's been commented out for 7 years; think it's from before we containerised the build/test process!